### ISSUE TITLE
JDBC Adapter Doesn't Quote Keywords

### ISSUE THREAD

**Issue by snehlsen** [EXASOL/virtual-schemas#40]
**Problem**
The Exasol JDBC Adapter for Virtual Schemas doesn't quote column names which are reserved keywords in the remote database. This causes queries to fail on the remote database, if the user selects a column named after a keyword but doesn't select all columns from the table (i.e. no SELECT * in the pushdown query). 
```sql
-- In the remote database:
CREATE TABLE "MONTH"(
"YEAR" VARCHAR(50) UTF8,
"MONTH" VARCHAR(50) UTF8,
YEAR_MONTH DATE);
-- In the local database:
ALTER VIRTUAL SCHEMA VS REFRESH;
SELECT "MONTH" FROM VS.MONTH;
```

**Comment by andrehacker** [comment]
Update: This is expected to be resolved in the next days.

**Comment by snehlsen** [comment]
Fix in v1.1.1
